packages/validator/src/commands/validateCmd.ts

```diff
@@ -1,23 +1,38 @@
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
 import * as utils from './utils';
 import * as logger from '../utils/logger';
 import { validate } from '../validator';
 import { Log, templates } from '../types';
 
-let template = process.env.npm_config_template || process.argv[2];
-const specs = process.env.npm_config_specs || process.argv[3];
-const version = process.env.npm_config_templateVersion || process.argv[4];
-
 const messages: Log[] = [];
 
+const args = yargs(hideBin(process.argv))
+  .option('template', {
+    description: 'Path to validator template file or name of airnode specification format',
+    default: '',
+    alias: 't',
+    string: true,
+  })
+  .option('specification', {
+    description: 'Path to specification file that will be validated',
+    default: '',
+    alias: ['specs', 's'],
+    string: true,
+  }).argv;
```

Comment:
It's a bit weired that you provide a default as an empty string and than you're checking whether it is provided. You can use [`demandOption`](https://github.com/yargs/yargs/blob/master/docs/api.md#demandoptionkey-msg) flag to require some option and let yargs handle it.

Reply:
@drgy This is still not resolved.